### ISSUE TITLE
Tutorial mode

### ISSUE THREAD

**Issue by coltonw** [coltonw/sensors-are-down#33]
Tutorial mode would automatically describe every card that is mentioned the first time it is mentioned that game.  Between games, players would be prompted to turn off tutorial mode.

**Comment by mturley** [comment]
Otherwise known as "verbose mode" maybe?  I like this idea, but I think maybe it should be made available on-demand mid-game when it's generally disabled, too.  Like, when prompted to make a decision, ask Alexa "explain more about what just happened" or "tell me about those cards" or whatever.

**Comment by coltonw** [comment]
Agreed.  I already have a "Describe" intent that is meant to be this, but it could definitely be improved (I don't think it describes enemy ships?)

**Comment by mturley** [comment]
I think a convenient alternative trigger phrase for this would be "what just happened?" because people might say that to themselves, not expecting a response :)
